src/App.jsx

```diff
@@ -14,70 +15,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const isAdded = (good) => {
+    setSelectedGood(good);
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const isDeleted = () => {
+    setSelectedGood('');
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood ? (
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood}
+          {' is selected'}
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={isDeleted}
+          />
+        </h1>
+      ) : (
+        <h1 className="title">No goods selected</h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isSelected = selectedGood === good;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                data-cy="Good"
+                className={isSelected ? 'has-background-success-light' : ''}
```

Comment:
consider to use [classnames](https://www.npmjs.com/package/classnames) library for conditionally joining classNames together. should be already exist in project.

```
import cn from 'classnames';
```

```suggestion
                className={cn({
                'has-background-success-light': isSelected,
              })}
```